llvm/lib/CodeGen/TargetPassConfig.cpp

```diff
@@ -1184,12 +1197,13 @@ bool TargetPassConfig::addISelPasses() {
   }
 
   if (YkInsertStackMaps) {
-    addPass(createYkStackmapsPass());
+    addPass(createYkStackmapsPass(numberOfControlPoints));
   }
 
   if (YkBasicBlockTracer) {
     addPass(createYkBasicBlockTracerPass());
   }
+  
```

Comment:
Remove this extra blank line.

Reply:
Removed :point_right: 1cf8ba2c198b168418677ebf9c9a4521e7b0b4e1